mobileversion.html

```diff
@@ -0,0 +1,186 @@
+        <section class="ton">
+            <h2 class="mps">Multi-Post<br>
+                Stories</h2>
+        </section>
+        <section class="sec">
+           <p class="mainp">
+           <h5 class="canopy">CANOPY</h5>
+           <p class="mid">&#8226;</p>
+           <p class="back">Back End Dev</p>
+           <p class="mid">&#8226;</p>
+           <p class="year">2015</p>
+           </p>
+        </section>
+        <section class="day">
+           <p class="daily">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+           </p>
+        </section>
+        <section class="maincon">
+           <ul>
+        <li class="container2"><a href="#">html</a></li>
+        <li class="container3"><a href="#">css</a></li>
+        <li class="container4"><a href="#">javaScript</a></li>
+        </ul>
+     </section>
+     <section class="see">
+        <a href="#" class="pro">See Project</a>
+     </section>
+     </div>
+     </div>
+     <div class="About">
+        <h2 class="Abouth2">About<br>
+            Myself</h2>
+            <section>
+                <p class="Hello">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </section>
+            <section>
+                <h3 class="Lets">Let's Connect</h3>
+            </section>
+            <section class="fonticons">
+                  <ul>
+                    <li><a href="#"><i class='fab fa-github'></i></a></li>
+                    <li><a href="#"><i class='fab fa-linkedin-in'></i></a></li>
+                    <li><a href="#"><i class='fab fa-angellist'></i></a></li> 
+                    <li><a href="#"><i class="fab fa-twitter"></i></a></li>
```

Comment:
:red_circle: **Your body should contain only 3 sections (headline, works, about). Note that the `<section> `tag means that the content inside relates to a single theme, and the `<div> `tag is used as a block part of the webpage and doesn't convey any particular meaning**
As stated by the previous reviewer the structure of your `<body>....</body> `should be
```
<body>
</header><nav>... </nav> </header>

<section>headline content <section>

<section>works content<section>

<section>About content<section>

</body>
```
**Apart from these 3 sections all the remaining sections tags should be replaced with `div` or `ul` in case you are dealing with a list of items**